README.md

```diff
@@ -450,8 +450,12 @@ The following sections describe the shape and behavior of these APIs in detail:
 
 When you require `react-native-code-push`, the module object provides the following top-level methods:
 
+* [allowRestart](#codepushallowrestart): Re-allows programmatic restarts to occur as a result of an update being installed, and optionally, immediately restarts the app if a pending update had attempted to restart the app while restarts were disallowed. This is an advanced API and is only neccessary if your app explicitly disallowed restarts via the `disallowRestart` method.
+
 * [checkForUpdate](#codepushcheckforupdate): Asks the CodePush service whether the configured app deployment has an update available. 
 
+* [disallowRestart](#codepushdisallowrestart): Temporarily disallows any programmatic restarts to occur as a result of a CodePush update being installed. This is an advanced API, and is useful when a component within your app (e.g. an onboarding wizard) needs to ensure that no end-user interruptions can occur during its lifetime.
```

Comment:
i did.

https://www.google.com/search?q=onboarding+wizard&ie=utf-8&oe=utf-8

unlike "watermark", i don't see any wikipedia or similar page describing what it means in a dictionary definition, just a bunch of articles where a couple people seem to describe their process as an "onboarding wizard" arbitrarily. if somebody, like me, who has never heard the term and doesn't know what it means, tries to look it up and its not immediately obvious to them what it is, then we should use a different term lol